### DISCUSSION TITLE
freeze with zebra module

### DISCUSSION THREAD

**Discussion by luzak34** [marcelotduarte/cx_Freeze#1699] [General]
I cant freeze my script with zebra module. After build i receive error:
File "C:\Users\bartek\AppData\Roaming\Python\Python310\site-packages\cx_Freeze\initscripts\__startup__.py", line 138, in run
    module_init.run(name + "__main__")
  File "C:\Users\bartek\AppData\Roaming\Python\Python310\site-packages\cx_Freeze\initscripts\console.py", line 16, in run
    exec(code, module_main.__dict__)  # pylint: disable=exec-used
  File "test.py", line 1, in <module>
  File "C:\Users\bartek\AppData\Roaming\Python\Python310\site-packages\zebra.py", line 25, in <module>
    import win32print
ImportError: DLL load failed while importing win32print: Nie można odnaleźć określonego modułu.

my script (simplified):

```
from zebra import Zebra
import win32print
z = Zebra()
printers=z.getqueues()

```
any ideas how to fix it?

**Comment by marcelotduarte** [comment]
Sorry for taking so long to see this.
Usually importing win32api or win32files together solves the problem.
Or via the command line using --includes=pywintypes
But I made a hook to make it work without hacks or options.

You can test the PR with:
`pip install --extra-index-url https://test.pypi.org/simple/ "cx-Freeze==8.5.3.3231.dev1767592263"`
Using uv:
`uv pip install --extra-index-url https://test.pypi.org/simple/ "cx-Freeze==8.5.3.3231.dev1767592263" --index-strategy unsafe-best-match`

**Reply by marcelotduarte** [reply]
[Release 8.6.0](https://github.com/marcelotduarte/cx_Freeze/releases/tag/8.6.0) is out!
[Documentation](https://cx-freeze.readthedocs.io)
